easy-coding-standard.yml

```diff
@@ -219,8 +219,9 @@ services:
 
 
 parameters:
-    skip_codes:
+    skip:
         SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff.UselessDocComment:
```

Comment:
We should add the UselessDocComment to the other coding standard, or am I missing something?

Reply:
I think you are right.